### PR TITLE
feat: Set tss metrics when constructing `TssBaseService`

### PR DESCRIPTION
**Pull request by petreze**
**Description**:
Construct metrics before initializing Hedera, so that we can pass a proper structure when initializing `TssBaseService`

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by mhess-swl — COMMENTED** [review]
All the changes look fine, but how does this PR accomplish actually initializing the tss metrics? (forgive me, it's EOB and I may have missed something obvious!)

File: hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java [hunk 1]
```diff
@@ -242,9 +260,6 @@ public static void main(final String... args) throws Exception {
         final var merkleCryptography = MerkleCryptographyFactory.create(configuration, cryptography);
         MerkleCryptoFactory.set(merkleCryptography);
 
-        // Register the metrics related to TSS functionalities
-        hedera.registerTssMetrics(metrics);
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java:246]: Some of these changes are merged in my PR also. You might have to merge back develop

Comment (petreze) [reply hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java:246]: Done

**Review by petreze — COMMENTED** [review]
Changed the title and the description accordingly after merging with develop. @mhess-swl

**Review by mhess-swl — APPROVED** [review]

**Pull request merged**